packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/ManageContest/steps/ContestLiveStep/ContestLiveStep.tsx

```diff
@@ -45,8 +49,14 @@ const ContestLiveStep = ({ createdContestAddress }: ContestLiveStepProps) => {
               />
               <CWButton
                 containerClassName="cta-btn"
-                label="Go to contests"
-                onClick={() => navigate('/manage/contests')}
+                label={
+                  isFarcasterContest ? 'Copy Farcaster Frame' : 'Go to contests'
+                }
+                onClick={() =>
+                  isFarcasterContest
+                    ? console.log('Farcaster frame copied')
```

Comment:
if this 'copy' logic from another pr?

Reply:
on click we need to copy the frame but this will be done in upcoming PRs